tendrl/gluster_integration/objects/volume/atoms/shrink/__init__.py

```diff
@@ -96,20 +96,46 @@ def run(self):
                             lock.acquire(lock_ttl=60)
                     except etcd.EtcdLockExpired:
                         continue
+                subvolumes = NS._int.client.read(
```

Comment:
Here you are reading bricks and variable is called subvolume. Subvolume is actually a set of bricks.